Sources/Fisticuffs/BidirectionalBindableProperty.swift

```diff
@@ -62,19 +62,34 @@ extension BidirectionalBindableProperty {
 //MARK: - Binding
 public extension BidirectionalBindableProperty {
     //MARK: Two way binding
-    func bind(_ observable: Observable<ValueType>) {
+    /// Bind property to observable
+    ///
+    /// - Parameters:
+    ///   - observable: The `Observable`
+    ///   - receiveOn: The `Scheduler` for the call back. Defaults to `MainThreadScheduler`
+    func bind(_ observable: Observable<ValueType>, receiveOn scheduler: Scheduler = MainThreadScheduler()) {
         bind(observable, DefaultBindingHandler())
     }
 
-    func bind<Data>(_ observable: Observable<Data>, _ bindingHandler: BindingHandler<Control, Data, ValueType>) {
+    /// Bind property to subscribable
+    ///
+    /// - Parameters:
+    ///   - observable: The `Observable`
+    ///   - bindingHandler: The custom `BindingHandler`
+    ///   - receiveOn: The `Scheduler` for the call back. Defaults to `MainThreadScheduler`
+    func bind<Data>(
+        _ observable: Observable<Data>,
+        _ bindingHandler: BindingHandler<Control, Data, ValueType>,
+        receiveOn scheduler: Scheduler = MainThreadScheduler()
```

Comment:
I think `receiveOn` should come before the `BindingHandler` argument. Most of the binding handlers use closures and it would be cleaner.

Reply:
I initially had that but this could be a breaking change. Also it won't make sense to have a `bind(_: , receiveOn: ..., _:)` pattern.

I can change if it we're good with that

Reply:
It would become a breaking change for sure if we add the argument names. But to me it doesn't make sense to have `receiveOn:` at the end because.

```
label.b_text.bind(someBindable, BindingHandlers.computed { x -> y in 
    // some long code
    return instanceOfY
}, receiveOn: SomeScheduler())
```

Reply:
Yea I agree.

@francocarbonaro what do you think? Should we go with this?

Reply:
I agree that `receiveOn` looks cleaner or at least it's more obvious when all properties appear before a closure.

Reply:
@ketenshi I agree, it's a breaking change but should be easy to fix.

Reply:
Updated. The `bind(_: receiveOn: _:)` still feels a bit awkward to me but I think it's a better change